sql-db/sql-app/src/test/java/io/quarkus/ts/openshift/sqldb/PostgresqlDevServicesIT.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.ts.openshift.sqldb;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.junit.DisabledOnNativeImage;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.DevModeQuarkusApplication;
+
+@QuarkusScenario
```

Comment:
I've proposed to name the tests using DevMode starting with `DevMode`. See https://github.com/quarkus-qe/quarkus-test-suite/pull/41/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R87.
What do you think?